BUILD.md

```diff
@@ -135,6 +133,12 @@ cd cactus
 npm run enable-corepack
 ```
 
+* Run this command to migrate lockfile (Also, run this everytime you have changes before running `yarn run cofugure`)
```

Comment:
@zondervancalvez Please go into a little more detail about the lockfile migration: explain that it is needed when someone was using yarn v1 but once it was migrated then there is no need. Also explain in a few words how to recognize which version it is (the lockfile has a version declaration in it somewhere IIRC)

> Also, run this everytime you have changes before running `yarn run cofugure`

Is this true though? If I change some .ts code and then want to run the configure script, what difference does the extra `yarn install` will make in that case in general if I already migrated my lockfile earlier?